hazelcast/src/main/java/com/hazelcast/internal/networking/nonblocking/SelectorMode.java

```diff
@@ -27,19 +27,22 @@
     SELECT_WITH_FIX;
 
     public static SelectorMode getConfiguredValue() {
-        return fromString(System.getProperty("hazelcast.io.selectorMode"));
+        return fromString(getConfiguredString());
+    }
+
+    public static String getConfiguredString() {
+        return System.getProperty("hazelcast.io.selectorMode", "select").trim().toLowerCase();
     }
 
     public static SelectorMode fromString(String value) {
-        String valueToCheck = StringUtil.isNullOrEmptyAfterTrim(value) ? null : value.trim().toLowerCase();
-        if (valueToCheck == null) {
+        if (value.equals("select")) {
             return SELECT;
-        } else if (valueToCheck.equals("selectnow")) {
-            return SELECT_NOW;
-        } else if (valueToCheck.equals("selectwithfix")) {
+        } else if (value.equals("selectwithfix")) {
             return SELECT_WITH_FIX;
+        } else if (value.equals("selectnow") || value.startsWith("selectnow,")) {
```

Comment:
`SELECT_NOW` is just used in 6 places. I would vote for a new constant `SELECT_WITH_IDLE_STRATEGY` instead of this hacky way. It doesn't look hard to implement.